docs_source/usage/vocabs-unique-keys.md

```diff
@@ -0,0 +1,128 @@
+# A Vocabulary for Identifying Uniqueness of Array Items
+
+## 1. Purpose
+
+This document describes a vocabulary defining keywords that can be used to validate that arrary instances contain unique values as determined by one or more given paths within each item.
+
+The intent for these keywords is to cover the use cases discussed in the JSON Schema vocabularies GitHub issue [#22](https://github.com/json-schema-org/json-schema-vocabularies/issues/22).
+
+## 2. Declarations
+
+The ID for this vocabulary is `https://gregsdennis.github.io/json-everything/vocabs-unique-keys` (the URI to this document).
+
+A draft 2020-12 meta-schema which includes this vocabulary has been defined for convenience.  The ID for the meta-schema is `https://gregsdennis.github.io/json-everything/meta/unique-keys`, and it can also be found at this address.
+
+## 3. The `uniqueKeys` Keyword
+
+### 3.1 Syntax and Semantics
+
+The value of `uniqueKeys` must be an array of JSON Pointers as defined by [RFC 6901](https://tools.ietf.org/html/rfc6901).  At least one JSON Pointer MUST be specified.
+
+These JSON Pointers, when resolved against each item, define a set of values for that item.  If an item does not contain the location specified by a JSON Pointer, that JSON Pointer is skipped and the resulting set does not contain a corresponding value for that JSON Pointer.
```

Comment:
> If an item does not contain the location specified by a JSON Pointer, that JSON Pointer is skipped and the resulting set does not contain a corresponding value for that JSON Pointer.

This may cause problems regarding positioning.  For example, given the pointer set

```
[ "/foo", "/baz", "/bar"]
```

this instance would fail

```json
[
    { "foo": 8, "bar": false },
    { "bar": false, "baz": 8 }
]
```

because both sets would contain `[8, false]` even though the `8` came from `foo` in the first item and `baz` in the second item.  It might make sense to have the implementation recognize a "missing" value, similar to how Javascript uses `undefined`.